docs/cli/overview.md

```diff
@@ -1,4 +1,4 @@
-# Overview
+# How it works
```

Comment:
Don't have a better name now, but it doesn't feel that what we do below is a How it works, at least, not in the sense of what we do in the explore API. It is more an introduction to functionalities provided in the CLI.

Reply:
I agree with Ivan. We're not showing how it works

Reply:
yes dont worry ill think another name